triggers/tests/daml/CreateAndExercise.daml

```diff
@@ -0,0 +1,43 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+-- Apache-2.0
+
+module CreateAndExercise where
+
+import DA.Next.Map (Map)
+import Daml.Trigger
+
+createAndExerciseTrigger : Trigger Int
+createAndExerciseTrigger = Trigger
+  { initialize = \_acs -> 3
+  , updateState = \_acs msg allowedFail -> case msg of
```

Comment:
The retries here look like you’ve copied them from another example but aren’t actually using them. Can we simplify the test case?

Reply:
I did and no I'm not. Indeed, I'll have a look.

Reply:
You can probably just change the trigger to `Trigger ()` and turn this into `\_ _ _ -> ()`

Reply:
Done.